quipucords/scanner/ansible/api.py

```diff
@@ -28,10 +36,19 @@ def from_connection_info(  # noqa: PLR0913
         :param username: The username to use for connecting to the server.
         :param password: The password to use for connecting to the server.
         :param ssl_verify: Whether to verify the SSL certificate.
+        :param proxy_url: proxy URL in the format 'host:port'.
         """
         base_uri = f"{protocol}://{host}:{port}"
         auth = HTTPBasicAuth(username=username, password=password)
-        return cls(base_url=base_uri, verify=ssl_verify, auth=auth)
+        session = cls(base_url=base_uri, verify=ssl_verify, auth=auth)
+
+        proxies = {}
+        if proxy_url:
+            proxies[protocol] = "http://" + proxy_url
```

Comment:
This will never have `https` protocol in the string?

Reply:
Also, micro optimization (assuming the http-only logic is valid:
 
```suggestion
            proxies[protocol] = f"http://{proxy_url}"
```

Reply:
Thank you for the explanation referencing the `requests` docs!